SpaceCore/Patches/GameLocationPatcher.cs

```diff
@@ -63,12 +63,12 @@ private static void After_Explode(GameLocation __instance, Vector2 tileLocation,
             SpaceEvents.InvokeBombExploded(who, tileLocation, radius);
         }
 
-        /// <summary>The method to call after <see cref="GameLocation.updateEvenIfFarmerIsntHere"/>.</summary>
-        private static void After_UpdateEvenIfFarmerIsntHere(GameLocation __instance, GameTime time)
-        {
-            // TODO: Optimize, maybe config file too?
-            __instance.terrainFeatures.Values.DoIf((tf) => tf is IUpdateEvenWithoutFarmer, (tf) => (tf as IUpdateEvenWithoutFarmer).UpdateEvenWithoutFarmer(__instance, time));
-            __instance.Objects.Values.DoIf((o) => o is IUpdateEvenWithoutFarmer, (o) => (o as IUpdateEvenWithoutFarmer).UpdateEvenWithoutFarmer(__instance, time));
-        }
+        ///// <summary>The method to call after <see cref="GameLocation.updateEvenIfFarmerIsntHere"/>.</summary>
+        //private static void After_UpdateEvenIfFarmerIsntHere(GameLocation __instance, GameTime time)
+        //{
+        //    // TODO: Optimize, maybe config file too?
+        //    __instance.terrainFeatures.Values.DoIf((tf) => tf is IUpdateEvenWithoutFarmer, (tf) => (tf as IUpdateEvenWithoutFarmer).UpdateEvenWithoutFarmer(__instance, time));
+        //    __instance.Objects.Values.DoIf((o) => o is IUpdateEvenWithoutFarmer, (o) => (o as IUpdateEvenWithoutFarmer).UpdateEvenWithoutFarmer(__instance, time));
+        //}
```

Comment:
Since there are no class that implement `IUpdateEvenWithoutFarmer`, these two loops -- executed _every tick_ -- will have no effect. So, I take this out to reduce lagging/stuttering.